tests/Unit/ExampleTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Tests\Unit;
+
+use PHPUnit\Framework\TestCase;
+
+class ExampleTest extends TestCase
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing doc comment for class ExampleTest](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)